relevanceai/vector_tools/spar.py

```diff
@@ -0,0 +1,61 @@
+from abc import abstractmethod
+import numpy as np
+import pandas as pd
+from rank_bm25 import *
```

Comment:
how do i install this

Reply:
I already slapped it to the requirements file